backend/app/Http/Controllers/AuthenticateController.php

```diff
@@ -290,7 +291,6 @@ public function verifyEmail(Request $request, $id, $token){
         $user = User::where("user_id", $id);
         $verifyToken = Token::where("token", $token);
 
-
         if($user->count() == 0){
             return $this->sendResponse(true, 'Invalid verification link, no user was found',"Invalid verification link", null, 404);
```

Comment:
please adjust this line to conform with the standard http response. thanks

Reply:
and add screenshots to any PR made.. thanks

Reply:
nothing has been done here.

Reply:
that's why it was given to you.. make it work. You should use your discretion to make it right becaus it is now your feature. I do not have right to write anything here or make changes because it was not allocated to me.